spec/bookingsync/api/client/rentals_contents_overrides_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+
+describe BookingSync::API::Client::RentalsContentsOverrides do
+  let(:client) { BookingSync::API::Client.new(test_access_token) }
+
+  before { |ex| @casette_base_path = casette_path(casette_dir, ex.metadata) }
+
+  describe ".rentals_contents_overrides", :vcr do
+    it "returns rentals contents overrides" do
+      expect(client.rentals_contents_overrides).not_to be_empty
+      assert_requested :get, bs_url("rentals_contents_overrides")
+    end
+
+    describe "links" do
+      it "returns associated rental" do
+        rentals_contents_override = client.rentals_contents_overrides.first
+        expect(rentals_contents_override.rental).not_to be_empty
+      end
+    end
+  end
+
+  describe ".rentals_contents_override", :vcr do
+    let(:prefetched_rentals_contents_override_id) {
+      find_resource("#{@casette_base_path}_rentals_contents_overrides/returns_rentals_contents_overrides.yml", "rentals_contents_overrides")[:id]
+    }
+
+    it "returns rentals_contents_override" do
```

Comment:
This only tests that the right endpoint is being requested, nothing about what's returned.
It's misleading when the `it` states it's testing something but it actually isn't.